sonar-plugin-api/src/main/java/org/sonar/api/server/rule/RulesDefinition.java

```diff
@@ -986,9 +994,18 @@ public DebtRemediationFunction debtRemediationFunction() {
       return debtRemediationFunction;
     }
 
+    /**
+     * @deprecated since 5.5, replaced by {@link #gapDescription()}
+     */
+    @Deprecated
     @CheckForNull
     public String effortToFixDescription() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)